lib/resolveGitRepo.js

```diff
@@ -19,6 +19,11 @@ const resolveGitConfigDir = async (gitDir) => {
   const stats = await fs.lstat(defaultDir)
   // If .git is a directory, use it
   if (stats.isDirectory()) return defaultDir
+  // If .git is a symbolic link, find realpath and use it,
+  // thought realpath is a directory, avoid infinite recursion.
+  if (stats.isSymbolicLink()) {
+    return fs.realpath(defaultDir);
```

Comment:
This probably needs to be wrapped in `normalizePath`:

```suggestion
    return normalizePath(await fs.realpath(defaultDir))
```